rules/stylistic_issues.js

```diff
@@ -90,5 +87,7 @@ module.exports = {
   "linebreak-style": ["error", "unix"], // https://github.com/dwyl/goodparts/issues/195
   "line-comment-position": "error", // https://github.com/dwyl/goodparts/issues/196
   "keyword-spacing": ["error", { "before": true, "after": true }], // https://github.com/dwyl/goodparts/issues/197
-  "key-spacing": ["error", { "beforeColon": false, "afterColon": true, "mode": 'strict' }] // https://github.com/dwyl/goodparts/issues/198
+  "key-spacing": ["error", { "beforeColon": false, "afterColon": true, "mode": 'strict' }], // https://github.com/dwyl/goodparts/issues/198
```

Comment:
prefer if settings were on _separate_ lines in the future to simplify tracking the diff when it gets updated...
(_but OK for now..._)